bin/pycbc_hdf5_splitbank

```diff
@@ -36,32 +36,32 @@ parser.add_argument("--version", action="version",
                   version=pycbc.version.git_verbose_msg)
 parser.add_argument("--bank-file", type=str,
                     help="Bank hdf file to load.")
-parser.add_argument('--templates-per-bank', type = int,
+parser.add_argument('--templates-per-bank', type=int,
```

Comment:
Since you want to introduce a `parser.add_mutually_exclusive_group()`, can we also include `--templates-per-bank` also in `outbanks` and do `outbanks = parser.add_mutually_exclusive_group(required=True)` since one of those options is always required? And then remove `if args.templates_per_bank and (args.number_of_banks or args.output_filenames):` in line 87 to raise the error message?

Reply:
OK - if you don't mind there being just a standard argparse error message about mutually exclusive options rather than a hand-crafted one.